book.rb

```diff
@@ -0,0 +1,9 @@
+# Represents the books class
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
```

Comment:
- After, adding the :rentals, let's also add its instance variable which you'd push to later.